### PR TITLE
punto17

### PR DESCRIPTION
**Pull request by jeffrey20122021**

### CODE REVIEW

File: cypress/page/iframe.page.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        return cy.iframe(this.iframePageTitle).find("h1");
+    }
+
+    goToCssPageInFrame() {
+        cy.iframe(this.iframePageTitle).find(this.cssBtn).click();
+    cy.frameLoaded(this.iframePageTitle, {url: this.cssPageURL});
+    }
+}
+
+export {IframePage};
```
Comment (holgiosalos) [inline cypress/page/iframe.page.ts:28]: EOF

File: cypress/page/iframe.page.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        cy.visit(this.pageURL);
+    }
+
+    getFrameTitle() {
+        return cy.iframe(this.iframePageTitle).find("h1");
+    }
+
+    goToCssPageInFrame() {
+        cy.iframe(this.iframePageTitle).find(this.cssBtn).click();
+    cy.frameLoaded(this.iframePageTitle, {url: this.cssPageURL});
```
Comment (holgiosalos) [inline cypress/page/iframe.page.ts:24]: fix the indentation

Comment (jeffrey20122021) [reply cypress/page/iframe.page.ts]: all done @holgiosalos  . In the terminal appear an error about CSS block . I was thinking if the URL is creating that problem.

**Review by holgiosalos — APPROVED** [review]

**Review by holgiosalos — COMMENTED** [review]
Please add the following property in the setupNodeEvents of the cypress.config.ts:

config.chromeWebSecurity = false;

That would fix the error, if not, you can modify cssPageURL to be:
`https://www.w3schools.com/css/`

**Pull request merged**